duckduckgo/duckduckgo-autofill
# Fix schema issue in extension repo

**Reviewer:** @sixers 
**Asana:** https://app.asana.com/0/0/1203648188543065/f

## Description
Fixes a type validation error when using a schema-generated type from the extension repo.

## Steps to test
@sixers From your extension branch, use this code and run tsc. You should no longer see those errors.
